packages/create-gasket-app/lib/scaffold/actions/load-preset.js

```diff
@@ -20,7 +20,7 @@ async function loadPresets({ context }) {
     packageManager: context.packageManager,
     dest: tmpDir
   });
-  const pkgVerb = pkgManager.isYarn ? 'add' : 'install';
+  const pkgVerb = pkgManager.manager ? 'add' : 'install';
```

Comment:
Do you mean to have
```suggestion
  const pkgVerb = pkgManager.manager === 'yarn' ? 'add' : 'install';
```

?

Reply:
What was wrong with `isYarn`?

Reply:
Ah yes 🤦‍♂️